digit-ui-esbuild/packages/modules/pgr/src/pages/employee/ComplaintDetails.js

```
@@ -351,8 +377,14 @@ export const ComplaintDetails = (props) => {
         setDisplayMenu(false);
         break;
       case "REOPEN":
-        setPopup(true);
-        setDisplayMenu(false);
+        const lastModifiedTime = complaintDetails?.service?.auditDetails?.lastModifiedTime;
```

Biome (2.5.3)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)